### PR TITLE
Fix - readme ♥ Add basic contribution guidelines

### PR DESCRIPTION
**Pull request by DDuran19**
Inayos ko lang ng onti ung guidelines - needs improvement in the future

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,39 +5,49 @@ Note by Director Mac: if gagawa ng project dapat nasa packages folder
...
-$ yarn build
+$ pnpm build
 ```
 
+# Directions for contributions
+
+1. Fork the repository from official FWDP Github
+2. Clone the repository to your local machine
+3. Make changes, follow github commit and pull conventions
+4. Go to github and createa a pull request
```
Comment (aishenreemo) [inline README.md]: typo oh @DDuran19

**Comment by directormac** [comment]
@aishenreemo Where is the typo.

**Comment by DDuran19** [comment]
saw it. will fix

**Review by directormac — CHANGES_REQUESTED** [review]
Please check typo's

**Review by aishenreemo — APPROVED** [review]
looks good

File: README.md (hunk 1 above)
Comment (DDuran19) [reply README.md]: Done fixing

**Review by DDuran19 — COMMENTED** [review]
Finished conversation

**Review by DDuran19 — COMMENTED** [review]
This is done

**Comment by aishenreemo** [comment]
galing galing naman oh choz

**Pull request merged**